Data/Vector/Unboxed.hs

```diff
@@ -1392,29 +1401,85 @@ product :: (Unbox a, Num a) => Vector a -> a
 product = G.product
 
 -- | /O(n)/ Yield the maximum element of the vector. The vector may not be
--- empty.
+-- empty. In a case of a tie the first occurrence wins.
+--
+-- ==== __Examples__
+--
+-- >>> import qualified Data.Vector.Unboxed as VU
+-- >>> import qualified Data.Vector.Generic as VG
+-- >>> VU.maximum $ VU.fromList [2.0, 1.0]
+-- 2.0
+-- >>> VG.maximum $ VU.fromList [2.0, 1.0]
+-- 2.0
```

Comment:
```suggestion
```
Seems like a copy paste error, right?

Reply:
Possibly. I don't want to dig through commit to find out, Fixed